apps/rig_api/test/rig_api/controllers/health_controller_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule RigApi.HealthControllerTest do
+  @moduledoc false
+  require Logger
+  use ExUnit.Case, async: true
+  use RigApi.ConnCase
+
+  describe "GET /health" do
+    test "should return OK as text response" do
+      conn = build_conn() |> get("/health")
+      assert conn.resp_body =~ "OK"
```

Comment:
You could use `==` here instead of `=~` since you expect the full body to match. If you'd like to assert that the response is actually a "text response", you could also assert the mime type to be `text/plain`.

Reply:
fixed - see latest commit